i18n/en.pot

```diff
@@ -356,6 +356,9 @@ msgstr ""
 msgid "Reserved values downloaded per TEI attribute"
 msgstr ""
 
+msgid "It is recommended a maximum of 500 reserve values"
```

Comment:
I think this message can be a bit shorter: `Recommended maximum is 500 reserve values`